src/monitoring_service/handlers.py

```diff
@@ -533,8 +532,8 @@ def action_monitoring_triggered_event_handler(event: Event, context: Context) ->
             )
         )
     except Exception as exc:  # pylint: disable=broad-except
-        error_message = exc.args[0]
-        if isinstance(exc, TransactionFailed) and "not allowed to monitor" in error_message:
+        error_message = exc.args[0] if len(exc.args) > 0 else ""
```

Comment:
I always wonder why there is no `list.get` method like there is for dicts. Would be nice, here.

Reply:
I am not aware of such a function. But yes, I would also love to have it. Do you know of any other solution. Haven't written much Python code in a long time.

Reply:
It's either your way or the "easier to ask forgiveness than permission" approach of
```python
try:
    error_message = exc.args[0]
except IndexError:
    error_message = ""
```
So no really elegant solution that I know of.

Reply:
Okay, than I would rather go with the current state if that is okay. It is only a single line and good enough to read. Nothing advanced.